packages/protocol/src/protocol.yml

```diff
@@ -1191,6 +1191,22 @@ BrowserContext:
       returns:
         writableStream: WritableStream
 
+    createTempDirectory:
```

Comment:
Let's merge this and previous methods and call it `createTempFiles`, giving an optional `rootDirName` parameter?